deploy/kubectl/openebs-nfs-provisioner.yaml

```diff
@@ -137,7 +137,7 @@ spec:
             command:
             - sh
             - -c
-            - test `pgrep -c "^provisioner-nfs.*"` = 1
+            - test `pgrep "^provisioner-nfs.*"` = 1
```

Comment:
should this be in helm chart as well?

Reply:
yes. Relevant PR https://github.com/openebs/dynamic-nfs-provisioner/pull/41